klaas/scripts/train_separation_model.py

```diff
@@ -14,6 +14,52 @@
 from ..feature_generation import feature_generation
 
 
+logging.basicConfig()
+log = logging.getLogger()
+
+
+def read_and_sample_data(path, key, columns_to_read, n_sample):
```

Comment:
Aren't those functions the same for each `train_script`? So they probably should go in a central place, like `klaas.io`